lsp/nls/src/server.rs

```diff
@@ -120,6 +138,9 @@ impl Server {
     pub fn run(&mut self) -> Result<()> {
         trace!("Running...");
         loop {
+            let never = crossbeam::channel::never();
+            let bg = self.background_jobs.receiver().unwrap_or(&never);
```

Comment:
Nitpick: given that `never` is used exactly once and is not very long, I would inline it. Feel free to ignore.

Reply:
Right, but it needs somewhere to live. Putting it inline makes it complain about borrowing a temporary.

Reply:
Ah, fair enough.